### DISCUSSION TITLE
PnP Modern Search V4 - Query Template Path:{\Site.URL}/<LibraryName> not working

### DISCUSSION THREAD

**Discussion by jameelrrs** [microsoft-search/pnp-modern-search#1767] [Q&A] [answered]
Hi,

To start with,  **PnP Modern Search WebPart is a great Solution**.  Amazing and appreciate the work done by the community.

What was working:
     Using a dynamic token in the PnP - Search Results **query template** to point my results to a specific library, since we have multiple sites as below:
    ` Path:{\Site.URL}/<LibraryName>`

Issue:
This was working fine until we planned to move to V4.  However the same query doesn't work in version V4.  Somehow, it adds \" in the beginning or ending of the queryModifier and adding LIbraryName after it doesn't give results.

Is there something missing.  Please suggest...  Thanks in advance.

**Comment by wobba** [comment]
I created a bug for this #1769.

**Reply by FranckyC** [reply]
Hi @jameelrrs, can you try with the `Path:{site.absoluteUrl}` token instead?

**Comment by FranckyC** [comment, marked as answer]
@jameelrrs Should be fixed in #1784.
